paparazzi/paparazzi-gradle-plugin/src/test/projects/view-with-lifecycle-usages/src/test/java/app/cash/paparazzi/plugin/test/LifecycleUsageTest.kt

```diff
@@ -0,0 +1,35 @@
+package app.cash.paparazzi.plugin.test
+
+import android.view.View
+import androidx.activity.OnBackPressedCallback
+import androidx.activity.findViewTreeOnBackPressedDispatcherOwner
+import app.cash.paparazzi.Paparazzi
+import org.junit.Rule
+import org.junit.Test
+
+class LifecycleUsageTest {
+  @get:Rule
+  val paparazzi = Paparazzi()
+
+  @Test fun OnBackPressedDispatcherOwner() {
+    val view = View(paparazzi.context)
+    view.doOnAttach {
+      val dispatcherOwner = checkNotNull(view.findViewTreeOnBackPressedDispatcherOwner()) {
+        "Could not find a OnBackPressedDispatcherOwner"
+      }
```

Comment:
let's remove this, as we own this test, and therefore don't need to check for this here.